apps/src/lib/node/ledger/ethereum_node/oracle/mod.rs

```diff
@@ -349,6 +361,22 @@ fn process_queue(
     confirmed
 }
 
+/// Functionality to do with publishing which blocks we have processed.
+pub mod last_processed_block {
```

Comment:
nit: module docstrings need to be moved inside the module
```suggestion
pub mod last_processed_block {
    //! Functionality to do with publishing which blocks we have processed.
```